src/dbt/kipptaf/models/kippadb/intermediate/int_kippadb__persistence.sql

```diff
@@ -57,7 +48,94 @@ select
             e.actual_end_date is null and eis.ugrad_status in ('Graduated', 'Attending')
         then 1
         else 0
-    end as is_persisting_fall,
+    end as is_persisting_int,
+
+    case
+        when date(r.academic_year, 10, 31) > current_date('{{ var("local_timezone") }}')
+        then null
+        when
+            e.actual_end_date >= date(r.academic_year, 10, 31)
+            and eis.ecc_account_id = e.school
```

Comment:
@cbini this is where the addition is used

Reply:
oh I see it now